packages/dropzone/test/dropzone.test.ts

```diff
@@ -87,17 +87,23 @@ describe('Dropzone', () => {
 
         expect(el.isDragged).to.be.false;
 
-        const dataTransfer = new DataTransfer();
-        const dragOverEvent = new DragEvent('dragover', {
-            dataTransfer,
-        });
-
-        el.dispatchEvent(dragOverEvent);
-
-        expect(el.isDragged).to.be.true;
-        // We should be able to make the following test here:
-        // expect(dataTransfer.dropEffect).to.equal(el.dropEffect);
-        // However, Chrome doesn't like it in the context of a test...
+        let dataTransfer: DataTransfer | boolean = false;
+        try {
+            // Safari doesn't like this...
+            dataTransfer = new DataTransfer();
```

Comment:
It'd be a little nicer if you could explicitly disable Safari for this test (right now if another browser broke, or someone made bad changes to this file, it has the potential to "pass" when it shouldn't.

Reply:
I'll check out whether there's an option for that in the runner. If not, we CAN add a `console.log()` in the `catch` so that it will at least report it to us...do you want to hold this PR or come back to this?

Reply:
Just saw your other comment, I think moving forward and then catching these two singularly will be the right call, thanks!